packages/edition-slices/src/tiles/tile-e/index.js

```diff
@@ -30,9 +40,11 @@ const TileE = ({ onPress, tile, breakpoint = editionBreakpoints.small }) => {
         style={styles.imageContainer}
         uri={crop.url}
         fill
+        hasVideo={hasVideo}
       />
       <TileSummary
         headlineStyle={styles.headline}
+        summary={getTileSummary(tile, withMoreTeaser ? 800 : 125)}
```

Comment:
```suggestion
        summary={getTileSummary(tile, 125)}
```
We aren't using TileE with `withMoreTeaser`, so we don't need that tile to support it.

Reply:
Is TileE not the one mentioned in the ticket? https://nidigitalsolutions.jira.com/browse/REPLAT-10562 , or should it only have a summary on mobile?

Reply:
It is the one. We just don't need withMoreTeaser functionality. 
For tablet we are using TileE for [support1](https://github.com/newsuk/times-components/blob/master/packages/provider-queries/src/section_fragment.graphql#L458).
Support1 doesn't have summary in the query. So there is no summary for TileE for tablet.
![image](https://user-images.githubusercontent.com/8720661/69252680-127a8d00-0bbc-11ea-96fd-23283c2b1fab.png)

Reply:
Sorry I misunderstood what you meant here but that makes sense, it doesn't ever have extended teaser content so there is no need for the prop or check. Updated the PR. :)